config.js

```diff
@@ -65,9 +66,13 @@ Config.prototype._seed = function _seed(seed) {
     // All config names should be camel-cased.
     seedOrDefault('TEST_KEY', 100, numValidator); // never remove, tests and lives depend on it
 
+    // Logger configs
+    seedOrDefault('defaultLogLevel', LoggingLevels.info);
+    seedOrDefault('gossipLogLevel', LoggingLevels.off);
+    seedOrDefault('joinerLogLevel', LoggingLevels.warn);
```

Comment:
reading 'joiner' makes me wonder -- is level setting intended to apply to objects (Joiner) or concepts (join)?

if the latter, then perhaps it should be s/joiner/join/ to match other join knobs?